atlp-rwanda/elite-bn-be
# feat(rating-accomodation): Users should be able to rate a centre

#### What does this PR do?
A users should be able to rate accommodation facilities in the centres that they have travelled to.
#### How should this be manually tested?
 1. Clone the Repo
 2. create a .env file and follow the .envExample
 3. run `npm i` , the `npm run dev` and `npm run test`  for testing endpoints
 Follow the following routes on postman or swagger to see if the endpoints work perfectly.

Test the following endpoints: 
`POST`:`api/v1/accomodation/rate`
`SWAGGER`: check on this link http://localhost:3000/api-docs/#/Users 
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201581
